### PR TITLE
chore(auto-chart):  add basic auto-chart package and demo use it

### PR DESCRIPTION
**Pull request by beewolf233**
### PR includes

- [ ] fixed #0
- [ ] add / modify test cases
- [ ] documents, demos

### CODE REVIEW

**Review by BBSQQ — CHANGES_REQUESTED** [review]
commit message should be "chore(auto-chart): xxxx"

**Review by BBSQQ — APPROVED** [review]
LGTM

File: packages/auto-chart/src/AutoChart.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  className?: string;
+  width?: number;
+  height?: number;
+  data?: any[];
+  toolbar?:boolean; 
+  development?: boolean;
+  title?: string;
+  description?: string;
+  purpose?: 'Comparison' | 'Trend' | 'Distribution' | 'Rank' | 'Proportion' | 'Composition';
+};
```
Comment (neoddish) [inline packages/auto-chart/src/AutoChart.tsx:14]: props like `toolbar` , `development` are not good named. we shall re-design these APIs.

leave a TODO tag here, or just remove props for now.

Comment (beewolf233) [reply packages/auto-chart/src/AutoChart.tsx]: already delete

**Review by neoddish — COMMENTED** [review]
LGTM

**Pull request merged**